Cakefile

```diff
@@ -434,3 +435,17 @@ task 'test:browser', 'run the test suite against the merged browser script', ->
   (-> eval source).call result
   testResults = runTests result.CoffeeScript
   process.exit 1 unless testResults
+
+task 'test:webpack', 'run webpack test', ->
+  args = [
+    "./node_modules/webpack/bin/webpack.js"
+    "--entry=./lib/coffeescript/browser.js", 
+    "--output-path=#{os.tmpdir()}",
+    "--output-filename=coffeescript.js"
```

Comment:
See https://github.com/webpack/docs/wiki/webpack-for-browserify-users#usage.

> webpack doesn't write to stdout. You need to specify a filename. It can't write to stdout because, unlike browserify, it is ready to generate multiple output files.